Makefile

```diff
@@ -4,6 +4,7 @@
 ifeq ($(GOPATH),)
 $(error The GOPATH environment variable needs to be set)
 endif
+GOPATH_PRIMARY := $(firstword $(subst :, ,${GOPATH}))
```

Comment:
This only works when building on Unix-like systems where colon is the path separator. For Windows we need to split on semicolon instead.